packages/documentation/src/content/docs/integration/requirements/assets.mdx

```diff
@@ -81,3 +80,150 @@ The `asset` object in the response will include the `tenantId` of the tenant to
 
   </TabItem>
 </Tabs>
+
+## Update an asset
+
+Once an asset has been created, you may need to adjust its operational threshold values. You can use the `updateAsset` mutation to modify the `withdrawalThreshold` and `liquidityThreshold`.
+
+These thresholds influence when Rafiki triggers low‑liquidity notifications but do not directly deposit or withdraw asset liquidity.
+
+For information about adding or removing liquidity, see [Asset liquidity](/admin/liquidity/asset-liquidity).
+
+<Tabs>
+  <TabItem label="Operation">
+    ```graphql
+    mutation UpdateAsset($input: UpdateAssetInput!) {
+      updateAsset(input: $input) {
+        asset {
+          id
+          code
+          scale
+          withdrawalThreshold
+          liquidityThreshold
+          tenantId
+        }
+      }
+    }
+    ```
+  </TabItem>
+
+  <TabItem label="Variables">
+    The input object for the update operation requires the asset `id`. Only the threshold fields are editable; all other fields are immutable.
+
+    ```json
+    {
+      "input": {
+        "id": "b3dffeda-1e0e-47d4-82a3-69b1a622eeb9",
+        "withdrawalThreshold": 100,
+        "liquidityThreshold": 500
+      }
+    }
+    ```
+
+    For more information about this mutation’s input object, see [`UpdateAssetInput`](/apis/graphql/backend/#definition-UpdateAssetInput).
+
+    :::note[Tenant ID and HMAC-signed request headers]
+    <TenantIdHmacNote />
+    :::
+
+  </TabItem>
+
+  <TabItem label="Response">
+    ```json
+    {
+      "data": {
+        "updateAsset": {
+          "asset": {
+            "id": "b3dffeda-1e0e-47d4-82a3-69b1a622eeb9",
+            "code": "USD",
+            "scale": 2,
+            "withdrawalThreshold": 100,
+            "liquidityThreshold": 500,
+            "tenantId": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
+          }
+        }
+      }
+    }
+    ```
+
+:::note[Tenant ID in the asset response]
+The `asset` object in the response will include the `tenantId` of the tenant to which the asset belongs. This `tenantId` is used to identify the tenant when processing requests related to the asset.
+:::
+
+  </TabItem>
+</Tabs>
+
+---
+
+## Delete an asset
+
+Deleting an asset removes it from active use within a Rafiki instance.
+
+You can only delete an asset if it's not associated with any peers or wallet addresses. Rafiki prevents deleting any asset that's still in use.
+
+:::danger
```

Comment:
Do you think we need the danger here? Given that it not possible to delete an asset with any existing references, I think admins shouldn't worry about breaking something :)